### ISSUE TITLE
Additional or excessive elements

### ISSUE THREAD

**Issue by michaldzwigala** [adshares/ads-operator#55]
Additional or excessive elements:

- [ ] TransactionKeyTransaction
+ Public_key: 9D46567A482F8F6AA567804EF2274F6ACEB370D8F7461C3A0DF4CE2C0DF432EB

- [ ] TransactionLogAccountTransaction:
-networkAccount

- [ ] TransactionSendManyTransaction:
     empty list
     - + wires: []

- [ ] TransactionStatusTransaction:
     + Status 8

- [ ] Additionally :
msgid [string] returns 8 and should be 0001: 00000008

**Comment by c3zi** [comment]
> msgid [string] returns 8 and should be 0001: 00000008

No, it shouldn't. It's a `msid` of the account which is used to make a transaction. In my opinion, the name is wrong, should be `account_msid` but we get this from ADS.
